fastlane/Fastfile

```diff
@@ -0,0 +1,55 @@
+  # This file contains the fastlane.tools configuration
+# You can find the documentation at https://docs.fastlane.tools
+#
+# For a list of all available actions, check out
+#
+#     https://docs.fastlane.tools/actions
+#
+# For a list of all available plugins, check out
+#
+#     https://docs.fastlane.tools/plugins/available-plugins
+#
+
+# Uncomment the line if you want fastlane to automatically update itself
+# update_fastlane
+
+default_platform(:android)
+
+platform :android do
+  appcenter_key = UI.input "Appcenter API Token: "
+
+  lane :release do | options |
+    debugVersion = UI.confirm "Is this a debug release? "
+    appcenter_version = appcenter_fetch_version_number(
+      api_token: appcenter_key
+    )
+    build_number = appcenter_version["build_number"]
+    increment_version_code(version_code: build_number)
+    versionNumber = appcenter_version["id"].to_i + 1
+    update_version_name(name: "Release #{versionNumber}")
+    gradle(task: "clean")
+    gradle(
+      task: "assemble",
+      build_type: debugVersion ? "Debug" : "Release",
+      properties: {
+        "archivesBaseName" => "sdkdemo_#{versionNumber}"
+      }
+    )
+    appcenter(notes: options[:notes])
+  end
+
+  lane :appcenter do | options |
+    apk_path = lane_context[SharedValues::GRADLE_APK_OUTPUT_PATH]
+    appcenter_upload(
+      api_token: appcenter_key,
+      owner_name: "Android-Demo",
+      app_name: "SDK-Demo",
+      file: apk_path,
+      destinations: UI.select("Select the distribution group for Appcenter: ", ["Collaborators", "Internal", "QA", "External"]),
```

Comment:
We'll never distribute to External, lets take this out so we don't end up distributing to whomever is in that group.